.github/workflows/release.yml

```diff
@@ -61,8 +61,9 @@ jobs:
 
       - name: Pack anvil-zksync
         run: |
-          tar -czf anvil-zksync-${{ inputs.tag || inputs.prerelease_name }}-${{ matrix.arch }}.tar.gz \
-            ./target/${{ matrix.arch }}/release/anvil-zksync
+          tar -C ./target/${{ matrix.arch }}/release -czf \
```

Comment:
Merge artifacts?

Reply:
Yea this comes from a merge commit. Would you prefer me to rebase instead? I remember weird behavior on Github when I rebased branches and then conversations tied to the old commits are impossible to resolve.

Reply:
Seems like you made https://github.com/matter-labs/anvil-zksync/pull/584 a part of your PR by accident